lib/plan_picker_web/templates/layout/root.html.eex

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8"/>
+    <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <title>Plan picker</title>
+    <link rel="stylesheet" href="<%= Routes.static_path(@conn, "/css/app.css") %>"/>
+    <link rel="icon" href="<%= Routes.static_path(@conn, "/favicon.svg") %>" />
+    <%= csrf_meta_tag() %>
+    <script defer type="text/javascript" src="<%= Routes.static_path(@conn, "/js/app.js") %>"></script>
+  </head>
+  <body>
+    <header>
+  <%= render "_navbar.html", assigns %>
+</header>
+  <main role="main" class="container">
+    <p class="alert alert-info" role="alert"><%= get_flash(@conn, :info) %></p>
+    <p class="alert alert-danger" role="alert"><%= get_flash(@conn, :error) %></p>
+      <%= @inner_content %>
```

Comment:
Liveview has access to `@socket` which will allow for adding alerts dynamically:
> The "app" and "live" layouts are often small and similar to each other, but the "app" layout uses the `@conn` and is used as part of the regular request life-cycle. The "live" layout is part of the LiveView and therefore has direct access to the `@socket`.

https://hexdocs.pm/phoenix_live_view/live-layouts.html

Reply:
I see, I will change that quickly.